packs/bit-bucket/actions/list_branches.py

```diff
@@ -0,0 +1,15 @@
+from lib.action import BitBucketAction
+
+
+class ListBrachesAction(BitBucketAction):
+    def run(self, repo):
+        """
+        List Braches of Repository with author names and message
+        """
+        bb = self.perform_request(repo=repo)
+        success, result = bb.get_branches()
+        branches = {}
+        for key, value in result.iteritems():
+            branches[key] = {'author': value['author'],
```

Comment:
Does the result object include more information about each branch?

If so, we should also include other available info.